packages/core-mobile/app/store/rpc/handlers/wc_sessionRequest/wc_sessionRequest.ts

```diff
@@ -200,7 +201,14 @@ class WCSessionRequestHandler implements RpcRequestHandler<WCSessionProposal> {
     const { proposer, requiredNamespaces, optionalNamespaces } = params
 
     const normalizedRequired = normalizeNamespaces(requiredNamespaces)
```

Comment:
can we add them to requiredNamespaces instead?

Reply:
I tried, but I got this error. 

![Simulator Screenshot - iPhone 15 Pro - 2024-08-13 at 12 31 34](https://github.com/user-attachments/assets/521ab3a3-7ce8-45cb-bf47-26cf639b685c)

so I guess this workaround only works with optional namespaces injected by the client.